app/views.py

```diff
@@ -901,6 +901,19 @@ def destroy(self, request, pk ):
         super(ArticleViewset, self).destroy(request,pk=pk)
 
         return Response(data={"success": "Article successfully deleted"})
+    
+    @action(methods=['get'], detail=True, url_path='comment/(?P<comment_id>.+)', permission_classes=[ArticlePermission])

```

Comment:
This function is not needed for now. As I have implemented the already implemented. Please dont include this function